src/material/datepicker/multi-year-view.ts

```diff
@@ -270,3 +305,8 @@ export class MatMultiYearView<D> implements AfterContentInit {
     return this._dir && this._dir.value === 'rtl';
   }
 }
+
+/** Gets remainder that is non-negative, even if first number is negative */
+export function euclideanModulo (a: number, b: number): number {
+  return (a % b + b) % b;
+}
```

Comment:
It seems to me that, currently, datepicker/public-api.ts is not exporting anything from ‘./multi-year-view’, so this function is already not introduced as API …